chainer/utils/conv_nd_kernel.py

```diff
@@ -0,0 +1,299 @@
+from __future__ import print_function
+import functools
+import six
+
+
+def identity(x):
+    return x
+
+
+def mulexp(xs, init=None):
+    if init:
+        return functools.reduce('{} * {}'.format, xs, init)
+    else:
+        return functools.reduce('{} * {}'.format, xs)
+
+
+def andexp(xs, init=None):
+    if init:
+        return functools.reduce('{} && {}'.format, xs, init)
+    else:
+        return functools.reduce('{} && {}'.format, xs)
+
+
+def muladdexp(xs, ys, init=None):
+    def aux(exp, arg):
+        x, y = arg
+        return '({} + {} * {})'.format(y, x, exp)
+    if init:
+        return functools.reduce(aux, zip(xs, ys), init)
+    else:
+        return functools.reduce(aux, zip(xs, ys))
+
+
+def maplist(fn, xs):
```

Comment:
`maplist` is called with `fn` being `identity`. If you do not plan to use this function any more, I think we should remove `fn` argument.